src/App.jsx

```diff
@@ -1,34 +1,95 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import * as todosAPI from './api/api';
 
-class App extends React.Component {
-  state = {
-    todos: [],
-    selectedUserId: 0,
+const App = () => {
+  const [todos, setTodos] = useState([]);
+  const [selectedUserId, setSelcetedUserId] = useState(0);
+  const [inputValue, setFilterValue] = useState('');
+  const [selectValue, setSelectValue] = useState('');
+  const [filteredTodos, setFilteredTodos] = useState([]);
+
+  useEffect(() => {
+    todosAPI.getTodos().then(res => setTodos(res));
+    todosAPI.getTodos().then(res => setFilteredTodos(res));
```

Comment:
why do you do this twice? one request is enough to set as many states as you wish

Reply:
fixed